README.md

```diff
@@ -63,7 +73,7 @@ $ bin/setup
 
 Run the tests
 ```
-$ rspec .
+$ RAILS_ENV=test rake db:drop db:create db:migrate; rspec
```

Comment:
You are telling that to run the tests it should drop and recreate the database all time. It is not true, tests should be run only with `rspec`.

What happened in your case is that you created the database with `bin/setup` and after, destroyed and recreated only the development one.

I think that is better to change
https://github.com/opensanca/opensanca_jobs/blob/master/bin/setup#L32

for:
```
system! 'bin/rails db:create db:migrate
```

and keep here in the doc only with `rspec`